examples/benchmarks/config_js.txt

```diff
@@ -22,3 +22,11 @@ effect_handlers_bench/product_early 32768
 effect_handlers_bench/resume_nontail 1366
 effect_handlers_bench/tree_explore 14
 effect_handlers_bench/triples 256
+input_output/dyck_one 5
```

Comment:
Do we want to use larger numbers here as well?

Reply:
Ah, yes, please adjust them such that each run takes between 0.5 and 2 seconds.